test/test_transformer.py

```diff
@@ -0,0 +1,93 @@
+"""Tests for Transformer."""
+
+from argparse import Namespace
+from typing import Iterator, Tuple
+
+import py
+import pytest
+
+from ansiblelint.rules import RulesCollection
+
+# noinspection PyProtectedMember
+from ansiblelint.runner import LintResult, _get_matches
+from ansiblelint.transformer import Transformer
+
+
+@pytest.fixture
+def copy_examples_dir(
+    tmpdir: py.path.local, config_options: Namespace
+) -> Iterator[Tuple[py.path.local, py.path.local]]:
+    """Fixture that copies the examples/ dir into a tmpdir."""
+    examples_dir = py.path.local("examples")
+    examples_dir.copy(tmpdir / "examples")
+    old_cwd = tmpdir.chdir()
+    config_options.cwd = tmpdir
+    yield old_cwd, tmpdir
+    old_cwd.chdir()
+
+
+@pytest.fixture
+def runner_result(
+    config_options: Namespace,
+    default_rules_collection: RulesCollection,
+    playbook: str,
+) -> LintResult:
+    """Fixture that runs the Runner to populate a LintResult for a given file."""
+    config_options.lintables = [playbook]
+    result = _get_matches(rules=default_rules_collection, options=config_options)
+    return result
+
+
+@pytest.mark.parametrize(
+    ("playbook", "matches_count", "transformed"),
+    (
+        # reuse TestRunner::test_runner test cases to ensure transformer does not mangle matches
+        pytest.param(
+            "examples/playbooks/nomatchestest.yml", 0, False, id="nomatchestest"
+        ),
+        pytest.param("examples/playbooks/unicode.yml", 1, False, id="unicode"),
+        pytest.param(
+            "examples/playbooks/lots_of_warnings.yml", 992, False, id="lots_of_warnings"
+        ),
+        pytest.param("examples/playbooks/become.yml", 0, True, id="become"),
+        pytest.param(
+            "examples/playbooks/contains_secrets.yml", 0, False, id="contains_secrets"
+        ),
```

Comment:
As we've reformatted all of the files, almost all of these tests are not actually transforming anything anymore. Only one is changing something now: in `become.yml` it changes the quotes on a string.